specs/derivation.md

```diff
@@ -798,24 +800,56 @@ To interact with the engine, the [execution engine API][exec-engine] is used, wi
 [exec-engine]: exec-engine.md
 
+##### Bedrock, Canyon, Delta: API Usage
+
 - [`engine_forkchoiceUpdatedV2`] — updates the forkchoice (i.e. the chain head) to `headBlockHash` if different, and
   instructs the engine to start building an execution payload if the payload attributes parameter is not `null`.
 - [`engine_getPayloadV2`] — retrieves a previously requested execution payload build.
 - [`engine_newPayloadV2`] — executes an execution payload to create a block.
 
-The current version of `op-node` uses the `v2` RPC methods from the engine API, whereas prior versions used the `v1`
-equivalents. The `v2` methods are backwards compatible with `v1` payloads but support Shanghai.
+##### Ecotone: API Usage
 
-[`engine_forkchoiceUpdatedV2`]: exec-engine.md#engine_forkchoiceupdatedv2
-[`engine_getPayloadV2`]: exec-engine.md#engine_getpayloadv2
-[`engine_newPayloadV2`]: exec-engine.md#engine_newpayloadv2
+- [`engine_forkchoiceUpdatedV3`] — updates the forkchoice (i.e. the chain head) to `headBlockHash` if different, and
+  instructs the engine to start building an execution payload if the payload attributes parameter is not `null`.
+- [`engine_getPayloadV3`] — retrieves a previously requested execution payload build.
+- `engine_newPayload`
+  - [`engine_newPayloadV2`] — executes a Bedrock/Canyon/Delta execution payload to create a block.
+  - [`engine_newPayloadV3`] — executes an Ecotone execution payload to create a block.
 
-The execution payload is an object of type [`ExecutionPayloadV2`][eth-payload].
+The current version of `op-node` uses the `v3` Engine API RPC methods as well as `engine_newPayloadV2`, due to
+`engine_newPayloadV3` only supporting Ecotone execution payloads. Both `engine_forkchoiceUpdatedV3` and
+`engine_getPayloadV3` are backwards compatible with Bedrock, Canyon & Delta payloads.
 
-[eth-payload]: https://github.com/ethereum/execution-apis/blob/main/src/engine/shanghai.md#payloadattributesv2
+Prior versions of `op-node` used `v2` and `v1` methods.
 
-With V2 of the execution payload, before Canyon the withdrawals field is required to be nil. After Canyon the
-withdrawals field is required to be non-nil. The op-node should set the withdrawals field to be an empty list.
+[`engine_forkchoiceUpdatedV2`]: exec-engine.md#engine_forkchoiceupdatedv2
+[`engine_forkchoiceUpdatedV3`]: exec-engine.md#engine_forkchoiceupdatedv3
+[`engine_getPayloadV2`]: exec-engine.md#engine_getpayloadv2
+[`engine_getPayloadV3`]: exec-engine.md#engine_getpayloadv3
+[`engine_newPayloadV2`]: exec-engine.md#engine_newpayloadv2
+[`engine_newPayloadV3`]: exec-engine.md#engine_newpayloadv3
+
+The execution payload is an object of type [`ExecutionPayloadV3`][eth-payload].
+
+[eth-payload]: https://github.com/ethereum/execution-apis/blob/main/src/engine/cancun.md
+
+The `ExecutionPayload` has the following requirements:
+
+- Bedrock
+  - The withdrawals field MUST be nil
+  - The blob gas used field MUST be nil
+  - The blob gas limit field MUST be nil
+- Canyon
```

Comment:
nit: `Canyon, Delta` instead of another line that says `Delta: exactly like Canyon`